portfolio/src/main/webapp/index.html

```diff
@@ -32,13 +32,16 @@ <h1>AnaK's Portfolio</h1>
 
       <p>Click <a href="placesGallery.html">here</a> to see some of my favourite places!</p>
 
-      <h1>Random Names</h1>
-      <p>Click here to see a random name:</p>
-      <button onclick="getRandomName()">Fetch a name!</button>
-      <h1 id="name-container"></h1>
+      <h1>Comments:</h1>
+      <ul id="comments-history"></ul>
+      
+      <form action="/data" method="POST">
+      <p>Write a new comment:</p>
+      <textarea name="new-comment"></textarea>
+      <br/><br/>
```

Comment:
nit: Do not close void elements like br or input.

i.e. <br>, not <br />

https://google.github.io/styleguide/htmlcssguide.html#Document_Type